bitnami/chainloop/Chart.yaml

```
@@ -0,0 +1,66 @@
+# Copyright Broadcom, Inc. All Rights Reserved.
+# SPDX-License-Identifier: APACHE-2.0
+
+annotations:
+  category: DeveloperTools
+  license: Apache-2.0
+  images: |
+    - image: docker.io/bitnami/chainloop-artifact-cas:0.95.3-debian-12-r0
+      name: artifact-cas
+    - image: docker.io/bitnami/chainloop-control-plane:0.95.3-debian-12-r0
+      name: control-plane
+    - image: docker.io/bitnami/chainloop-control-plane-migrations:0.95.3-debian-12-r0
+      name: control-plane-migrations
+    - image: docker.io/bitnami/dex:2.40.0-debian-12-r1
+      name: dex
+apiVersion: v2
+appVersion: 0.95.3
+dependencies:
+  - name: common
+    repository: https://charts.bitnami.com/bitnami
+    tags:
+      - bitnami-common
+    version: 2.x.x
+  - condition: postgresql.enabled
+    name: postgresql
+    repository: https://charts.bitnami.com/bitnami
+    version: 15.x.x
+  - condition: development
+    name: vault
+    repository: https://charts.bitnami.com/bitnami
+    version: 1.4.x
```

Subcharts must use the bitnami OCI registry
```suggestion
dependencies:
  - name: common
    repository: oci://registry-1.docker.io/bitnamicharts
    tags:
      - bitnami-common
    version: 2.x.x
  - condition: postgresql.enabled
    name: postgresql
    repository: oci://registry-1.docker.io/bitnamicharts
    version: 15.x.x
  - condition: development
    name: vault
    repository: oci://registry-1.docker.io/bitnamicharts
    version: 1.4.x
```